config/kubernetes/sig-release/teams.yaml

```diff
@@ -303,11 +274,11 @@ teams:
               as a notification group. Remove org members who are not current
               Release Team Leads.
             members:
-            - hasheddan # 1.20 RT Lead Shadow
-            - jeremyrickard # 1.20 RT Lead
-            - lachie83 # 1.20 Emeritus Adviser
-            - palnabarun # 1.20 RT Lead Shadow
-            - savitharaghunathan # 1.20 RT Lead Shadow
+            - bai # 1.21 RT Lead Shadow
+            - oikiki # 1.21 Emeritus Adviser
```

Comment:
Hi Nabarun, Can you replace 1.21 Emeritus Adviser with 1.21 RT Lead Shadow?

Reply:
Also, I think Kirsten's GH id is @kikisdeliveryservice

Reply:
Done! 3AM commits are harmful. xD

Reply:
I got a promotion!! Thank you for your confidence!!!! :wink: